src/config.test.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   CODE_VERSION: 'test',
   DEPLOYED_ENV: 'test',
   REACT_APP_ARCHIVE_URL: '/apps/archive/codeversion',
+  SEARCH_URL: 'https://search-os-f42d77d.sandbox.openstax.org',
```

Comment:
@TomWoodward search isn't working in the browser tests. I thought adding this might help but no dice